code/database.py

```diff
@@ -0,0 +1,87 @@
+from sqlalchemy import create_engine, Column, Integer, Boolean, String, DateTime
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+import datetime
+import hashlib
+from FileSystemObject import File, Directory
+
+
+
+
+engine = create_engine('sqlite:///SyncOrSwimDB.db')
```

Comment:
These lines of code are not part of any class. Should these be in the Database class?